routes/production_lines.js

```diff
@@ -176,4 +195,19 @@ router.post('/delete/:id', function(req, res, next) {
   });
 })
 
+/**
+ * TODO
+ * Req 7.3.3
+ * Managers should be able to select a production and mark it completed, viewing the resulting lot number (req 7.2.7) and causing the product to enter the appropriate inventory (req 7.2.8 for intermediates and 7.2.9 for final products).
+ * @param  {[type]} req   [description]
+ * @param  {[type]} res   [description]
+ * @param  {[type]} next) {             } [description]
+ * @return {[type]}       [description]
+ */
+router.post('/mark_completed/:id', function(req, res, next) {
```

Comment:
@dy46 Marking production lines as complete.

Reply:
if you want you can do a unique id generator for lot numbers, right now i'm just getting a random number between two large numbers, which isn't super good